diff --git a/node_modules/fhenixjs/lib/esm/sdk/fhe/fhe-browser.js b/node_modules/fhenixjs/lib/esm/sdk/fhe/fhe-browser.js
index 13baee2..6f5a824 100644
--- a/node_modules/fhenixjs/lib/esm/sdk/fhe/fhe-browser.js
+++ b/node_modules/fhenixjs/lib/esm/sdk/fhe/fhe-browser.js
@@ -7,7 +7,7 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
         step((generator = generator.apply(thisArg, _arguments || [])).next());
     });
 };
-import wasm from "./tfhe_bg.wasm";
+import wasm from "./tfhe_bg-browser.wasm?url";
 //@ts-expect-error imports from different wasm/js files are wonky
 import initSDK from "./tfhe.js";
 let initialized;
